### DISCUSSION TITLE
信息汇总到这里有用吗？

### DISCUSSION THREAD

**Discussion by liuwenwu520** [WeNeedHome/SummaryOfLoanSuspension#659] [Q&A]
信息汇总到这里有用吗？国家能看到吗？这里还充斥着五毛，话题已经被搅混了。

**Comment by hktk1024** [comment]
汇总到国务院?
国务院比你清楚

**Comment by cccp057** [comment]
有用啊，你自己其实可以根据数量估算一下规模嘛，也可以给吃瓜群众提供参考嘛

**Reply by cccp057** [reply]
做研究可以根据数量和地理分布写论文，记者可以根据项目找住户写深度报道，吃瓜群众可以看看本地烂尾楼的情况看那些企业不靠谱，烂尾风险大不大，怎么没用？

所谓话题搅混，无非就是说一些人借机闹事辱骂中国嘛，虽然删不了号，那你可以骂回去嘛

**Comment by C-Joey** [comment]
你说的是国家还是政府 看不懂你指的是哪个 
这边汇集信息又不是给政府看的 是给烂尾楼业主和其他关心的人看的
政府当然实时监控这个项目

**Comment by yangtuqiang** [comment]
不放墙外，在墙内等着封么

**Comment by liuwenwu520** [comment]
这个事情说实话相比起来只是少数，这些业主也无法团结，我已经看不到这方面的报道了，估计这次是解决不了问题的，这次疫情又卷土重来，经济如果倒退的话，应该会爆出更多的问题，到那个时候才是真正解决问题的时候。

**Reply by NiceAir** [reply]
是的，现在还不够，还没到全面爆雷的时候。消息封锁自古以来都有，但是历朝历代中人民被压榨到一个点之后终究会爆发。光想压下去不想解决终究会搬起石头砸自己的脚

**Reply by imhappyok** [reply]
还爆发傻，这是21世纪，汉人早已没有秦汉时期的血性，几千年的愚民政策导致很多人习惯了忍受不公平的待遇，00后接受的西方文化多一些，可能他们有希望改变吧，其实政府收集剩余财富是可以理解的，但是不能割的太狠了，老百姓生活没有保障，迟早会出乱子

**Comment by xiaer93** [comment]
行动才能带来改变，虽然很难，但是请保持对美好生活的向往，也注意保护好自己。

**Reply by purpose1113** [reply]
做总比什么也不做好

**Comment by imhappyok** [comment]
2022.7.19是315套
2022.7.20是319套
2022.7.21还是319套
感觉战斗力不行啊，满打满算也不到100w户，预估应该在50w户左右，政府压下来不难，这场火应该要是熄灭了，各位还是老老实实的继续还贷吧，反抗不了还是默默忍受，别折腾了

**Comment by jiushi555** [comment]
没有用，但我们不能忘
